integration/backward_compatibility/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -x
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../lib/common.bash"
+
+RUNTIME=${RUNTIME:-kata-runtime}
+RUNTIME_URL=https://github.com/kata-containers/runtime
+
+# returns the latest stable version of kata-runtime
+install_latest_stable() {
+	latest="$(git ls-remote --tags "${RUNTIME_URL}"  | egrep -o "refs.*" | egrep -v "\-alpha|\-rc|{}" | cut -d/ -f3 | sort -r | head -1)"
+	"${cidir}/../../.ci/install_kata.sh" "${latest}"
+}
+
+main() {
+	runtime_path=$(get_docker_kata_path ${RUNTIME})
+	if [ -z "${runtime_path}" ] || [ "${runtime_path}" = "null" ]; then
+		die "${RUNTIME} has not been configured as a runtime in docker"
+	fi
+
+	# run a container
+	cont_name="backward_test"
+	current="$(${runtime_path} --version)"
+	docker run -dti --name "${cont_name}" --runtime="${RUNTIME}" debian bash
```

Comment:
Nit: the `busybox` image should already be available which might make this test faster to run?